.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+name: "build"
+on: [push, pull_request]
+env:
+  TRIVY_VERSION: 0.43.1
+  COPA_VERSION: 0.3.0
+permissions: write-all
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup BATS
+        uses: mig4/setup-bats@v1
+        with:
+          bats-version: 1.7.0
+
+      - name: Check out code
+        uses: actions/checkout@v1
```

Comment:
v1 is fairly old, can you update to latest (v3.5.3)? let's follow up to add dependabot and secure-repo to automate updating these